src/Sylius/Bundle/AdminBundle/Resources/views/Currency/_form.html.twig

```diff
@@ -0,0 +1,7 @@
+{{ form_errors(form) }}
+<div class="one field">
+    {{ form_row(form.code) }}
+    {{ form_row(form.exchangeRate) }}
+    {{ form_row(form.enabled) }}
+</div>
+{{ form_rest(form) }}
```

Comment:
Do we need this? After [last changes](https://github.com/Sylius/Sylius/pull/4656) ``{{ form_row(form._token) }}`` and ``{{ form_end(...) }}`` are rendered as default.